File: PHP/Cpu.php
<?php
class Cpu{
    
    private $nama; //deklarasi Attribute
    private $harga; //deklarasi Attribute

    #constructor
    function __construct($nama, $harga){
        $this->nama = $nama;
        $this->harga = $harga;

    }
    #set get
    function setNama($nama){
        $this->nama = $nama;
    }
    function getNama(){
        return $this->nama;
    }
    function setHarga($harga){
        $this->harga = $harga;
    }
    function getHarga(){
        return $this->harga;
    }
    
    #keluaran
    function printCpu(){
        echo "---- CPU ----". "<br>";
        echo "Name: " .$this->nama."<br>";
        echo "Price: " .$this->harga."<br>";
    }

    function __destruct()
    {
        
    }
}
?>

File: Python/Cpu.py
class Cpu():

    #Constructor 
    def __init__(self):
        
        self.nama = "" #deklarasi atribut
        self.harga = 0 #deklarasi atribut
    
    #set get
    def setNama(self, nama):
        self.nama = nama
    def getNama(self):
        return self.nama
    def setHarga(self, harga):
        self.harga = harga
    def getHarga(self):
        return self.harga
   
    #keluaran
    def printCpu(self):
        print("---- CPU ----")
        print("Name: " + str(self.nama))
        print("Price: " + str(self.harga))